Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Multi-stage Dockerfile, to build and package an extraction plugin
```

Comment:
Nice to see the `Dockerfile` reintroduced in the template. This gives plugin developers a lot more flexibility. It allows different base images, extra libraries etc.

Reply:
The template never had a `Dockerfile`, so it is the first introduction. :)
It was never intended to not have a `Dockerfile` when writing a plugin.

Reply:
It was there in the very first commit it seems :-). But that might have been very short-lived indeed.